src/SFML/Audio/Music.cpp

```diff
@@ -119,7 +118,8 @@ bool Music::onGetData(SoundStream::Chunk& data)
     data.sampleCount = static_cast<std::size_t>(m_file.read(&m_samples[0], m_samples.size()));
 
     // Check if we have reached the end of the audio file
-    return data.sampleCount == m_samples.size();
+    return (data.sampleCount == m_samples.size()) &&
+           (m_file.getSampleOffset() < m_file.getSampleCount());
```

Comment:
Looking through SFML's code I think we don't do line breaks for lines of the given length.